crates/bevy_derive/src/bevy_main.rs

```diff
@@ -11,7 +11,7 @@ pub fn bevy_main(_attr: TokenStream, item: TokenStream) -> TokenStream {
     TokenStream::from(quote! {
         #[no_mangle]
         #[cfg(target_os = "android")]
-        unsafe extern "C" fn ANativeActivity_onCreate(
```

Comment:
`no_mangle` is inherently unsafe - I think it's useful to have this unsafe as a marker of that.

Also, I'm fairly sure that there are requirements on the arguments to make calling this safe, i.e. exactly the same requirements as `ndk_glue::init` https://github.com/rust-windowing/android-ndk-rs/blob/e8765fed70c45351c3df2ea19d19c98d8a8073a3/ndk-glue/src/lib.rs#L139-L141

See e.g. https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=0b0b590a6f7eda7f89453affce714391 which demonstrates why this would need to be here